reference/carts.v3.yml

```diff
@@ -1551,16 +1552,16 @@ components:
                         valueId:
                           type: number
                           description: The product option value identifier.
```

Comment:
I would consider adding, "in number format." and possibly provide an example of a value id